setup/static/render.js

```diff
@@ -2,17 +2,21 @@ $(document).ready(function() {
 	if ($("#webmail").val() == 'none') {
 		$("#webmail_path").hide();
 		$("#webmail_path").attr("value", "");
+        $("#webmail_host").parent().hide();
```

Comment:
For styling purposes i would suggest to keep the same indentation here. Beside that looks good

Reply:
Sorry was using spaces instead of tabs. Fixed :smiley: